internal/config/config.go

```diff
@@ -44,6 +44,10 @@ type Config struct {
 	KafkaGroupID            string
 	KafkaInventoryTopic     string
 	KafkaSystemProfileTopic string
+	KafkaUsername			string
```

Comment:
Please sort these alphabetically. It makes it easier to manage the list of config options.